pandas/tests/io/test_parquet.py

```diff
@@ -765,6 +765,10 @@ def test_timestamp_nanoseconds(self, pa):
         check_round_trip(df, pa, write_kwargs={"version": "2.0"})
 
     def test_timezone_aware_index(self, pa, timezone_aware_date_list):
```

Comment:
should be outside but temporarily is ok

note we actually do want to pin pyarrow in some older builds but i'll comment on the issues